src/entity/Source.ts

```diff
@@ -165,4 +165,7 @@ export class SquadSource extends Source {
 
   @Column({ default: 0 })
   memberInviteRank: number;
+
+  @Column({ default: true })
```

Comment:
Is it not default false per RFC as well?

Reply:
https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/1151500290/RFC-069+-+Admin+approval+on+Squads+posts#Table

Reply:
Yeah, this should be `false` to be backwards compat.

Reply:
> https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/1151500290/RFC-069+-+Admin+approval+on+Squads+posts#Table



> Is it not default false per RFC as well?

@capJavert I made this change in response to your comment on my previous PR. Although looking back at it now, I understand that I misunderstood your comment, which is the entire reason this error originated in the first place! 🤦🏼

Reply:
😆 Oh I see it now, yeah I was referring to the fact that we will need a script to update and `moderationRequired=true` will just be sent from frontend of course.

Reply:
Ah I guess it doesn't really mater what the default is right?
It will be based on user input + state of other field anyway, we can just do this for backward compat and then ensure we set it right for the new incoming requests.